uproot/tree.py

```diff
@@ -423,6 +423,9 @@ def array(self, branch, interpretation=None, entrystart=None, entrystop=None, fl
 
     def arrays(self, branches=None, outputtype=dict, namedecode=None, entrystart=None, entrystop=None, flatten=False, cache=None, basketcache=None, keycache=None, executor=None, blocking=True):
         branches = list(self._normalize_branches(branches))
+        if flatten is None:
+            branches = [(branch, interpretation) for branch, interpretation in branches if not isinstance(interpretation, asjagged)]
+            flatten = False
```

Comment:
None evaluates to False, so line 428 is not required.

Reply:
Yeah, but as I said before, I don't rely on that in my internal code. When I say `if x`, I manually ensure that `x` is boolean. This is a style choice that may or may not agree with a PEP, but I'm sticking to it (and enforcing it in my projects) because I think it's safer.

Reply:
That's fine. Thanks for the fix!